src/Microsoft.DotNet.Wpf/src/Common/src/Interop/Windows/Kernel32/Interop.FreeLibrary.cs

```diff
@@ -0,0 +1,14 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    internal static partial class Kernel32
+    {
+        [DllImport(Libraries.Kernel32, SetLastError = true, ExactSpelling = true)]
```

Comment:
Now `GeneratedDllImport` is renamed to `LibraryImport` and generally available to third party code. We should consider to use it like BCL.

Reply:
Since this repo does not use `LibraryImport` yet, I don't want this large PR to be the first to use it. I think it would be best to do this in a smaller PR or in a PR the only migrates `DllImport` to `LibraryImport`.